pkg/infrastructure/platform/platform.go

```diff
@@ -62,6 +64,9 @@ func ProviderForPlatform(platform string, fg featuregates.FeatureGate) (infrastr
 	case powervstypes.Name:
 		return terraform.InitializeProvider(powervs.PlatformStages), nil
 	case openstacktypes.Name:
+		if fg.Enabled(configv1.FeatureGateClusterAPIInstall) {
+			return clusterapi.InitializeProvider(openstackcapi.Provider{}), nil
+		}
```

Comment:
Can you also add this to the platform_altinfra.go file?

I recently added a little background on the altinfra image in the enhancement: https://github.com/openshift/enhancements/blob/56e53e353b0201936b687e4f0d1ada3897ddf201/enhancements/installer/bootstrapping-clusters-with-capi-providers.md#test-plan